x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -167,7 +166,7 @@ async function claimAvailableTasks(opts: TaskClaimerOpts): Promise<ClaimOwnershi
   }
 
   // apply limited concurrency limits (TODO: can currently starve other tasks)
-  const candidateTasks = applyLimitedConcurrency(currentTasks, batches);
+  const candidateTasks = selectTasksByCapacity(currentTasks, batches);
```

Comment:
wonder if we still need this since we're searching directly using the msearch?

Reply:
We discussed offline and given the code works with concurrency of `1`, we can follow up the work to fix the code when concurrencies > 1 https://github.com/elastic/kibana/issues/191301